dev-manuals/architecture/chickenshock/README.md

```diff
@@ -0,0 +1,46 @@
+# Architecture of Chickenshock
+
+Project: https://github.com/Gamify-IT/chickenshock \
+Scripts: https://github.com/Gamify-IT/chickenshock/tree/main/Assets/Scripts
+## Purpose
+
+Being a fun-to-play minigame that lets users choose one correct answer from a bunch of wrong answers
+
+## General code structure
+
+Unity uses C# scripts.
+There is no "main"-class in Unity scripts. 
+Each script that is put on a `GameObject` can have a `Start` method (called once when the `GameObject` is created), an `Awake` method (called once before `Start`) and an `Update` method (called on each frame). 
+If the `GameObject` is destroyed everything attached to it will be destroyed aswell, meaning no script attached to the destroyed `GameObject` will be executed again.
```

Comment:
```suggestion
Unity uses C# scripts. \
There is no "main"-class in Unity scripts. \
Each script that is put on a `GameObject` can have a `Start` method (called once when the `GameObject` is created), an `Awake` method (called once before `Start`), and an `Update` method (called on each frame). \
If the `GameObject` is destroyed everything attached to it will be destroyed as well, meaning no script attached to the destroyed `GameObject` will be executed again.
```